ref-exchange/src/owner.rs

```diff
@@ -35,6 +35,8 @@ impl Contract {
     /// Migration function from v2 to v2.
     /// For next version upgrades, change this function.
     #[init(ignore_state)]
+    // [AUDIT_09]
+    #[private]
```

Comment:
@referencedev Just to confirm that we call `migrate` in the same receipt as we deploy new contract.

Reply:
It's done in the same call, yes. See code below of `upgrade` where it's initiated.

Reply:
So, we are good here, right?